src/database-layer/db.ts

```diff
@@ -293,7 +293,7 @@ if (maybePg != null) {
 					map: (iterator, thisArg) => rows.map(iterator, thisArg),
 				},
 				rowsAffected: rowCount,
-				insertId: _.get(rows, '0.id'),
+				insertId: _.get(rows, [ 0, 'id' ]),
```

Comment:
Would it not be even quicker to not rely on a function call at all?

Reply:
Possibly but it'd be a lot messier and only save a couple of % at most I expect, as opposed to the ~94% improvement of this change